### ISSUE TITLE
In-page link targets need tabindex

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#5670]
## Package
_Choose from the list:_
- [x] app-content-pages
- [ ] app-project
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Describe the bug
Category links on the publications and team pages send focus to the document body, breaking the page for keyboard navigation and screen readers. 

https://www.tpgi.com/how-to-avoid-breaking-web-pages-for-keyboard-users/#videodemo

## Expected behavior
The category heading should be focused after navigating to that heading. This can be fixed by adding `tabindex` to link targets (see the linked post from TPGi.)
